### PR TITLE
Minimum player count for nuclear emergency

### PR DESCRIPTION
**Pull request by flappybatpal**
## About the PR 
Adds a 15 minimum ready player count to the nuclear mode.
Adds a build define to skip game mode checks, replaces THE_REVOLUTION_WILL_NOT_BE_TELEVISED.

## Why's this needed? 
Low player counts isn't ideal for team modes.
Nuclear can start with just two players right now.
15 players suggested by Gannets.

## Changelog 

```changelog
(u)Flappybat
(*)The Nuclear Emergency game mode now requires 15 players.
```

### CODE REVIEW

**Review by Gannets — APPROVED** [review]

File: _std/__build.dm [hunk 1]
```diff
@@ -47,6 +47,7 @@ o+`        `-` ``..-:yooos-..----------..`
 //#define BONUS_POINTS // gives a bunch of starting points to various abilities/uplinks/weapon vendors
 //#define SHUT_UP_AND_GIVE_ME_MEDAL_STUFF // causes has_medal to always return true - good for testing medal rewards etc.
 //#define SHOW_ME_STATUSES // incredibly hacky visible status effects
+#define ME_AND_MY_40_ALT_ACCOUNTS // Override game mode minimum player requirements for testing revs, nukies etc.
```
Comment (pgmzeta) [inline _std/__build.dm:50]: This should be commented out by default, right?

File: code/datums/gamemodes/nuclear.dm [hunk 2]
```diff
@@ -48,13 +48,21 @@
 
 		if (player.ready)
 			num_players++
+#ifndef ME_AND_MY_40_ALT_ACCOUNTS
+	if (num_players < 15)
```
Comment (pgmzeta) [inline code/datums/gamemodes/nuclear.dm:52]: this should be an in-file define like `AMOUNT_OF_VALID_NUKE_PLANT_LOCATIONS`; make sure to reference the value in the output message.

File: code/datums/gamemodes/nuclear.dm [hunk 3]
```diff
@@ -48,13 +48,21 @@
...
+		boutput(world, SPAN_ALERT("<b>ERROR: Minimum player count of 15 required for Nuclear game mode, aborting nuke round pre-setup.</b>"))
```
Comment (pgmzeta) [inline code/datums/gamemodes/nuclear.dm:53]: recommend logging this instead; the game will re-roll mode automatically now iirc?

**Review by pgmzeta — CHANGES_REQUESTED** [review]

File: _std/__build.dm (hunk 1 above)
Comment (flappybatpal) [reply _std/__build.dm:50]: GUESS WHO WAS TESTING IT AND FORGOT

File: code/datums/gamemodes/nuclear.dm (hunk 2 above)
Comment (flappybatpal) [reply code/datums/gamemodes/nuclear.dm]: Making a var, I don't see a good reason for a define

**Review by pgmzeta — APPROVED** [review]

**Pull request merged**